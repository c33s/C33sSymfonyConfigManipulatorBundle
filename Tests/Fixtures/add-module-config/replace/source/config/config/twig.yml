# This configuration does not contain YAML, so it can be overwritten
#twig:
#    debug:            "%kernel.debug%"
#    strict_variables: "%kernel.debug%"
