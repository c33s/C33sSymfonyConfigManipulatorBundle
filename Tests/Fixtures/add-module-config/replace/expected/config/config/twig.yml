twig:
    foo: bar
    bar: baz
